src/util/AutoFetchMore.ts

```diff
@@ -0,0 +1,10 @@
+import { MutableRefObject } from "react";
+
+export function AutoFetchMore(
+  ref: MutableRefObject<HTMLDivElement | undefined | null>,
+  onLoadMore?: () => void,
+) {
+  if (ref.current && ref.current?.scrollHeight <= document.documentElement.clientHeight) {
+    onLoadMore?.();
+  }
+}
```

Comment:
Since they are related, can you move this and `handleScroll` into the same file? Since they both relate to scrolling, the file could be called scroll.ts.
Then, please rename them to be more appropriate revolving around scrolling. 
ex.
handleScroll -> onScrollToBottom
AutoMergeFetch -> autoFillPage

These names aren't great so if you can think of better names for them, that are general, please do. Or, one thing I can't decide is whether these components are too specific to infinite scroll. If so, we could move them both into an infinite-scroll.ts file in the Common directory. What do you think?

Reply:
I think infinte-scroll.ts is suitable as I don't think we can use it any where else